integrations/discord-bot/voxguard/cogs/automod_cmds.py

```
@@ -0,0 +1,234 @@
+        )
+
+    @automod.command(name="allow-domain", description="Allow a domain past the links rule.")
+    @require_operator()
+    async def automod_allow_domain(
+        self, interaction: discord.Interaction, domain: str, remove: bool = False
+    ) -> None:
+        runtime = self.bot.runtime
+        config = runtime.config(interaction.guild.id)
+        entry = config["automod"]["rules"].setdefault("links", {})
+        allowed = {d.lower() for d in entry.get("allowed_domains", [])}
+        clean = domain.lower().strip().lstrip("www.")
```

Ruff (0.15.21)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)